LabController/src/bkr/labcontroller/watchdog.py

```diff
@@ -152,7 +152,15 @@ def poll(self):
         # Start a new monitor for any active watchdog we are not already monitoring.
         for watchdog in active_watchdogs:
             if watchdog['recipe_id'] not in self.monitor_greenlets:
-                self.spawn_monitor(watchdog)
+                try:
+                    self.spawn_monitor(watchdog)
```

Comment:
So, now, you return `[]` in get_console_files when system_name is not defined. 
Why do we need this change. We should be covered.

So what kind of error are we trying to cover here? I don't believe that Exception should be placed here. It is a broad exception and we may ignore important errors.

Reply:
Code has since been removed.  This was the hotfix so we could bring up beaker production.  The only difference was the log message wasn't as detailed as I wrote here.